### PR TITLE
MCH CLUSTER MAPS

### PR DESCRIPTION
**Pull request by victorvalenciatorres**

### CODE REVIEW

File: Detectors/MUON/MCH/Contour/include/MCHContour/SVGWriter.h [hunk 1]
```diff
@@ -72,7 +72,7 @@ class SVGWriter
   template <typename T>
   void polygon(const o2::mch::contour::Polygon<T>& p)
   {
-    mSVGBuffer << R"(<polygon points=")";
+    mSVGBuffer << R"(<polygon fill="blue" points=")";
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Contour/include/MCHContour/SVGWriter.h]: I guess this is a leftover from a test. Please remove it, as there's no point in hard coding a color here.

File: Detectors/MUON/MCH/Evaluation/CMakeLists.txt [hunk 2]
```diff
@@ -33,3 +33,8 @@ o2_target_root_dictionary(MCHEvaluation
                             include/MCHEvaluation/Draw.h
                           LINKDEF src/MCHEvaluationLinkDef.h)
 
+o2_add_executable(map_mch
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Evaluation/CMakeLists.txt]: `map_mch` is not a terribly good name :wink:
Especially considering the executable name will be `o2-mch-map_mch` which stutters on Mch and does not respect the executable naming convention.

What about something along the lines of `clustermap-display` or `clustermap-plotter` ? @mguilbau any opinion here ?

File: Detectors/MUON/MCH/Evaluation/README.md [hunk 3]
```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
 $ root
 root[0] o2::mch::eval::drawAll("compare.root")
 ```
+# MCH Cluster Maps
+
+The general purpose is to track "unexpected" detector issues not well reproduced with MC simulations. These problems generate non-negligible bias in Acc*Eff corrections resulting in large tracking systematic uncertainties.
+
+During the data reconstruction, the status of the detector is calculated with the CCDB which is used to discard most of the detector issues.
+This status map is built with information based on pedestals, high voltage tension, occupancy etc.
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Evaluation/README.md]: high voltage is not currently included in the computation of the statusmap. It's planned, but not yet done.

File: Detectors/MUON/MCH/Evaluation/README.md [hunk 4]
```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
...
+
+Nevertheless, some detector issues (e.g. a cable swapping) are not well detected online and consequently not properly reproduced by the CCBD.
+The main objective of this code is to spot these issues not included in the status map.
+
+Input files used:
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Evaluation/README.md]: I guess the documentation on how to use this program should be primarily seen from the point of the view of the next person that will _use it_ to produce cluster map comparisons, not so much develop the code. So how to get the input files should be part of the documentation for instance. (by the way that doc could be put in a separate `.md` file to keep the main README.md more as an index.

File: Detectors/MUON/MCH/Evaluation/README.md [hunk 5]
```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
...
+MC_QC.root
+o2sim_geometry-aligned.root
+
+Src file:
+map_mch.cxx
+
+COMPILATION (commands):
+
+cd alice ==> alienv enter O2/latest
+cd sw/BUILD/O2-latest/O2 ==> cmake --build .
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Evaluation/README.md]: this is kind of an expert way of working and is only relevant for people developing the code, not simply using it, so I'd remove it from this doc.

File: Detectors/MUON/MCH/Evaluation/README.md [hunk 6]
```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
...
+
+cd InputFiles ==> o2-mch-map_mch --green --normperarea --rootfileleft DATA_QC.root --rootfileright MC_QC.root
+
+HELP MESSAGE (command):
+
+o2-mch-map_mch --help
+
+OPEN OUTPUT FILES:
+
+cd outputfile ==> open CHAMBERS-1-NB.html CHAMBERS-2-NB.html CHAMBERS-3-NB.html CHAMBERS-4-NB.html CHAMBERS-5-NB.html CHAMBERS-6-NB.html CHAMBERS-7-NB.html CHAMBERS-8-NB.html CHAMBERS-9-NB.html CHAMBERS-10-NB.html
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Evaluation/README.md]: I believe the `open` command is a Mac-only thing, so not reproducible e.g. on a Linux machine. Either change it to plain English (open in your preferred browser the CHAMBERS*.html files generated) or provide the equivalent Linux command...

**Comment by aphecetche** [comment]
will review the "meat" of the PR (the actual map_mch code) later on...

File: Detectors/MUON/MCH/Evaluation/CMakeLists.txt (hunk 2 above)
Comment (mguilbau) [reply Detectors/MUON/MCH/Evaluation/CMakeLists.txt]: Clustermap-display is my preferred

**Pull request closed without merging**